.github/workflows/test-and-package.yml

```
@@ -1,9 +1,9 @@
-name: Build
+name: Test and Package
```

```suggestion
name: Maven Build
```
Again, Timur's suggested name conveys what happens well and concisely.